### ISSUE TITLE
Wondering if toggle is called multiple times.

### ISSUE THREAD

**Issue by toger5** [Calinou/fps-test#3]
https://github.com/Calinou/fps-test/blob/4b8ca178722bcab1cab9374bad5e593a50d6895c/scripts/player.gd#L42

wouldn't that be called for every input?
Example:
mouse moves + mouse button pressed + whatever_key_is_set_up_as_toggle pressed calls _input() 3 times. and since whatever_key_is_set_up_as_toggle is pressed it goes through the if statement 3 times.

**Comment by Calinou** [comment]
Indeed, `event.is_action_pressed("toggle_mouse_capture")` should have been used instead. This is a really old project, it will eventually need to be remade from scratch using Godot 3.0 (I learned a lot about the engine since I've made this demo).

**Comment by toger5** [comment]
Okay good to know. I was wondering if I misunderstand something...
Because I was running into issues. Th even function was not called for each event.
than I realised that my computer is not moving the mouse when pressing a key (stupid palm detection...) -> so there was no event.

Your script was still helpful by the way !!

Should I close or do you want to keep it as a reminder to redo it for 3.0 ? or update it one day?

**Comment by Calinou** [comment]
> Should I close or do you want to keep it as a reminder to redo it for 3.0 ? or update it one day?

It can be kept opened for now.
